storage_service/Dockerfile

```diff
@@ -14,8 +14,8 @@ RUN apt-get install -y --no-install-recommends \
     && rm -rf /var/lib/apt/lists/*
 
 COPY --from=builder /usr/src/app/target/release/c2pa_storage_service /usr/local/bin/
-COPY --from=aquasec/trivy:latest /usr/local/bin/trivy /usr/local/bin/trivy
-RUN trivy filesystem --exit-code 1 --no-progress /
+#COPY --from=aquasec/trivy:latest /usr/local/bin/trivy /usr/local/bin/trivy
```

Comment:
why are we disabling trivy?

Reply:
it crashes the container, because of some missing files. I think we shouldn't be running it within the Dockerfile anyway, and instead maybe scan the container in a separate workflow once a week or so.

Reply:
tracked in #49